test/plugin/scenarios/mongodb-3.x-scenario/bin/startup.sh

```diff
@@ -18,4 +18,4 @@
 
 home="$(cd "$(dirname $0)"; pwd)"
 
-java -jar ${agent_opts} ${home}/../libs/mongodb-3.x-scenario.jar &
\ No newline at end of file
+java -jar -Dskywalking.plugin.mongodb.trace_param=true ${agent_opts} ${home}/../libs/mongodb-3.x-scenario.jar &
```

Comment:
You activated this, but can't see the param in the tags.

Reply:
param is db.statement,In terms of content judgment, I have encountered a problem, can you help me see it, for example, 
![image](https://user-images.githubusercontent.com/16585330/93715651-2c4e7d00-fb9d-11ea-8a52-975b7733f328.png)
I can't see the difference between the estimated value and the actual value, but it is an error.

Reply:
What do you mean `error`? Why CI reports success?

Reply:
> param is db.statement,In terms of content judgment

Also, this place seems like a bug. The parameter should be another tag, like other plugins. Could you fix them too? And if no parameter exists, the tag should not be there too.

Reply:
> What do you mean `error`? Why CI reports success?

This pr uses not null judgment, I think it is better to use string equality judgment.

Reply:
Let's adjust the tag and don't worry about this check for now, OK?

Reply:
Wait, is the statement including the entity name of the operation or not?

Reply:
From my reading from your screenshot, only operation is a part of the statement, I think the plugin code has some issue about assembling the statement and parameter. Statement should be an expression and parameters are variable. I am not sure whether mongodb is friendly to this?

Reply:
I just read the codes again, the statement tag should be removed, as this actually isn't a statement anymore. We should use `DB_BIND_VARIABLES` tag to replace `DB_STATEMENT`, because the statement is being used to check slow SQL. Also, please help to remove the `executeMethod` as the prefix, it has been the operation name of the span.

Reply:
> From my reading from your screenshot, only operation is a part of the statement, I think the plugin code has some issue about assembling the statement and parameter. Statement should be an expression and parameters are variable. I am not sure whether mongodb is friendly to this?

Mongo does not seem to have a placeholder similar to sql.

Reply:
> I just read the codes again, the statement tag should be removed, as this actually isn't a statement anymore. We should use `DB_BIND_VARIABLES` tag to replace `DB_STATEMENT`, because the statement is being used to check slow SQL. Also, please help to remove the `executeMethod` as the prefix, it has been the operation name of the span.

Okay, let me adjust.